fineract-provider/src/main/resources/sql/migrations/core_db/V27__add-loan-type-column-to-loan-table.sql

```diff
@@ -19,48 +19,3 @@
 
 ALTER TABLE `m_loan`
     ADD COLUMN `loan_type_enum` SMALLINT NOT NULL AFTER `loan_status_id`;
-
--- create collateral management table
-CREATE TABLE IF NOT EXISTS `m_collateral_management` (
```

Comment:
Hmm... why are we deleting the lines relating to collateral management tables? Is this in the right PR?

Reply:
Nope. This collateral module is related to another PR and added the queries to a new file